fonduer/parser/parser.py

```diff
@@ -262,28 +262,53 @@ def _parse_figure(self, node, state):
         :param state: The global state necessary to place the node in context
             of the document as a whole.
         """
-        if node.tag != "img":
+        if node.tag not in ["img", "figure"]:
             return state
 
         # Process the figure
         stable_id = "{}::{}:{}".format(
             state["document"].name, "figure", state["figure"]["idx"]
         )
 
+        # img within a Figure get's processed in the parent Figure
+        if node.tag == "img" and isinstance(state["parent"][node], Figure):
+            return state
+
+        # NOTE: We currently do NOT support nested figures.
         if not isinstance(state["parent"][node], Section):
             logger.warning("Figure is nested within {}".format(state["parent"][node]))
-            raise NotImplementedError("Nested Figure not implemented.")
+            return state
 
-        # Create the Figure entry in the DB
-        state["context"][node] = Figure(
-            document=state["document"],
-            section=state["parent"][node],
-            stable_id=stable_id,
-            position=state["figure"]["idx"],
-            url=node.get("src"),
-        )
-        state["figure"]["idx"] += 1
+        # If processing a raw img
+        if node.tag == "img":
+            # Create the Figure entry in the DB
+            state["context"][node] = Figure(
+                document=state["document"],
+                section=state["parent"][node],
+                stable_id=stable_id,
+                position=state["figure"]["idx"],
+                url=node.get("src"),
+            )
+        elif node.tag == "figure":
+            # Pull the image from a child img node, if one exists
+            imgs = [child for child in node if child.tag == "img"]
+
+            if len(imgs) > 1:
+                logger.warning("Figure contains multiple images.")
+                raise NotImplementedError("No support for figures with multiple <img>")
```

Comment:
Right now, this just crashes. If we want to be more robust, a better answer might be to basically strip the `<figure>` tag and process all the included images individually. Not sure what we'd do about the a `<figcaption>` if one was present. Perhaps just ignore it.